test/SaveGraphTest.php

```diff
@@ -25,31 +24,8 @@ public function testReturnsTrueOn204()
         $handler = HandlerStack::create($mock);
         $guzzle = new Client(['handler' => $handler]);
         $api = new FedoraApi($guzzle);
-        $client = new Chullo($api);
 
-        $result = $client->saveGraph("", new \EasyRdf_Graph());
-        $this->assertTrue($result);
-    }
-
-    /**
-     * @covers            Islandora\Chullo\Chullo::saveGraph
-     * @uses              GuzzleHttp\Client
-     */
-    public function testReturnsFalseOtherwise()
-    {
-        $mock = new MockHandler([
-            new Response(412),
-            new Response(409),
-        ]);
-
-        $handler = HandlerStack::create($mock);
-        $guzzle = new Client(['handler' => $handler]);
-        $api = new FedoraApi($guzzle);
-        $client = new Chullo($api);
-
-        foreach ($mock as $response) {
-            $result = $client->saveGraph("", new \EasyRdf_Graph());
-            $this->assertFalse($result);
-        }
+        $result = $api->saveGraph(new \EasyRdf_Graph());
+        $this->assertEquals(204, $result->getStatusCode());
```

Comment:
Same question [as above](https://github.com/Islandora-CLAW/chullo/pull/70/files#r112701471), do we generate a resource on an empty graph or do we require some content before we persist it to the repository?